lib/deliver/commands_generator.rb

```diff
@@ -36,7 +36,8 @@ def run
         c.action do |args, options|
           options = FastlaneCore::Configuration.create(Deliver::Options.available_options, options.__hash__)
           loaded = options.load_configuration_file("Deliverfile")
-          loaded = true if options[:description] || options[:ipa] # do we have *anything* here?
+          options.verify_conflicts
```

Comment:
Hey @KrauseFx, I think that this verification should be implemented on `fastlane_core` level and `verify_conflicts` should be called each time we load options values(from CLI, config file or setting defaults).
I added a few updates to [conflicting options PR](https://github.com/fastlane/fastlane_core/pull/82) wrt.

Thanks again for highlighting it!